server/index.js

```diff
@@ -37,84 +37,44 @@ app.use(express.static(PUBLIC_PATH));
 app.use(express.static(DIST_PATH));
 
 io.on('connection', socket => {
+  Axios.put(`http://localhost:3000/api/user/socketConnect/${socket.id}`);
   socket.on('message', async data => {
     try {
-      const { chatroomId, author, message, userId } = data;
+      const { chatroomId, author, message, recipient, title } = data;
       const newMessage = await ChatMessage.create({
         chatroomId,
         author,
         message,
-        userId,
+        recipient,
       });
       const chatroom = await Chatroom.findByPk(chatroomId);
-      const users = chatroom.chatusers.split('/').filter(id => id !== userId);
-      await users.forEach(async id => {
-        const alert = await Alert.create({
-          subject: `New Message Received From ${author} in ${chatroom.name}`,
-          userId: id,
-        });
-        const user = await User.findOne({
-          include: [{ model: Session }],
-          where: {
-            id,
-          },
-        });
-        if (user) {
-          user.sessions.forEach(session => {
-            io.to(session.socket).emit('alert', alert);
-            io.to(session.socket).emit('newMessage', newMessage);
+      await chatroom.update({
+        [title]: chatroom[title] + 1,
+      });
```

Comment:
I'm pretty lost as to what's going on with this title thing and why we're adding 1 to it, but lines 50 and 51 could be done in 1 query--
await Chatroom.update({ [title]: chatroom[title] + 1 }, { where: { id: chatroomId }})